### PR TITLE
fix: add excludedCourseRoles to bulk-grades fetch

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 

reworked some badly-located filter logic that was preventing the excludedCourseRoles key from making it into bulk-grades download.
JIRA: [AU-76](https://openedx.atlassian.net/browse/AU-76)

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**
Load a test with staff members.  download bulk grades both With and Without including course members.  Verify that both have data, and that the staff members are missing from file intended to exclude them.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
Tested locally, looks good 👍

**Review by jansenk — APPROVED** [review]

**Review by mattcarter — APPROVED** [review]
checked locally. Approved 👍
